### ISSUE TITLE
Adding new item to children example (App3) throws

### ISSUE THREAD

**Issue by gaearon** [chenglou/react-state-stream#4]
I added "Add new" button to the third example with the following handler:

```js
  handleAddClick: function () {
    var items = this.state.items,
        lastItem = items[items.length - 1] || 'a';

    items.push(String.fromCharCode(lastItem.charCodeAt(0) + 1));
    items.sort();

    this.setState({
      items: items,
    });
  },
```

It throws with undefined `config` on [this line](https://github.com/chenglou/react-state-stream/blob/master/App3.jsx#L104).

I don't really understand Mori or lazy streams yet so I'd appreciate if you helped me figure out the issue.

**Comment by gaearon** [comment]
It seems like `configs` are only computed once in `getInitialStateStream` so they don't take new children into account.

**Comment by chenglou** [comment]
Yeah it's very manual right now. I'll abstract the whole logic away into a component/mixin soon.
